### DISCUSSION TITLE
Referring to Pins on ATTiny1614

### DISCUSSION THREAD

**Discussion by ghost** [SpenceKonde/megaTinyCore#1065] [Q&A] [answered]
Using Arduino IDE 2.3.2 I had a problem referring to Port B pins on an ATTiny1614; all the Port A pins worked using e.g. A2 but using B0 and B1 flashed am LED on A4 and A5 respectively, and B2 and B3 caused a compiler error.  So I read the section 'How to Refer to Pins' in the README.md file and learned that I should use PIN_PXn format i.e. PIN_PB0, which cured all the problems.  I have not heard of this nomenclature before and would like to know if it is applicable to any MCU using the Arduino IDE platform, or just the development tools in megaTinyCore?

**Comment by hmeijdam** [comment, marked as answer]
You van find the PIN_PXn definitions in the "variants" folder of the core.  There they are listed for the 2, 4, 6 and 7 series in a file called "pins_arduino.h"

**Reply by ghost** [reply]
Yes thank you I have found them.  I did try using PIN_PA2 in a Nano sketch resulting in a compiler error so I guess this nomenclature only applies to ATTinys and megaTinyCore.
